shipment_advice/views/shipment_advice.xml

```diff
@@ -187,12 +187,15 @@
                     <field name="loaded_package_ids" invisible="1" />
                     <notebook>
                         <page
-                            string="Lading"
+                            string="Loading"
```

Comment:
You will need to discuss with @jgrandguillaume about that, the term `Lading` is on purpose :)

Reply:
It did feel weird with loading everywhere below, anyway I dropped the commit.